dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/PutJob.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * ****************************************************************************
+ *    Copyright 2014-2017 Spectra Logic Corporation. All Rights Reserved.
+ *    Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *    this file except in compliance with the License. A copy of the License is located at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    or in the "license" file accompanying this file.
+ *    This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *    CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *    specific language governing permissions and limitations under the License.
+ *  ****************************************************************************
+ */
+package com.spectralogic.dsbrowser.gui.services.jobService
+
+import com.google.common.collect.ImmutableMap
+import com.spectralogic.ds3client.helpers.Ds3ClientHelpers
+import com.spectralogic.ds3client.metadata.MetadataAccessImpl
+import com.spectralogic.dsbrowser.api.services.logging.LogType
+import com.spectralogic.dsbrowser.gui.services.jobService.data.JobData
+import com.spectralogic.dsbrowser.gui.services.jobService.stage.PrepStage
+import com.spectralogic.dsbrowser.gui.services.jobService.stage.TeardownStage
+import com.spectralogic.dsbrowser.gui.services.jobService.stage.TransferStage
+import com.spectralogic.dsbrowser.gui.services.jobService.util.ChunkWaiter
+import com.spectralogic.dsbrowser.gui.services.jobService.util.Stats
+import io.reactivex.Completable
+import io.reactivex.Observable
+import io.reactivex.disposables.Disposable
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+class PutJob(private val putJobData: JobData) : JobService(), PrepStage<JobData>, TransferStage, TeardownStage {
+
+    private var job: Ds3ClientHelpers.Job? = null
+    private var chunkWaiter: ChunkWaiter = ChunkWaiter()
+    private var stats: Stats = Stats()
+    private var log: Logger = LoggerFactory.getLogger(PutJob::class.java)
```

Comment:
Can some of these be made `val`?  Also, the logger should be updated to be consistent with the suggested change above.